docs/timeseries.md

```diff
@@ -10,7 +14,7 @@ Accept headers: `application/json`, `text/csv`, `application/geo+json`
 
 Possible header and request parameter combinations:
 
-- If the Accept header is `text/csv` or `applicatin/geo+json`, the raw CSV or GeoJSON response is returned.
+- If the Accept header is `text/csv` or `application/geo+json`, the raw CSV or GeoJSON response is returned.
 - If the Accept header is `application/json` with an output field of `geojson`, the entire JSON object with metadata including GeoJSON response is returned.
 - If the Accept header is `application/json` with an output field of `csv`, the entire JSON object with metadata including CSV response is returned.
 - If the Accept header is `application/json` without an output field, the entire JSON object with metadata including GeoJSON response is returned.
```

Comment:
Does it make sense to add another line for what the 'default' behavior is when there are no Accept headers but an output format is included? Or to indicate inline if any of these are the same as default if you don't include the header? I think that would be the entire JSON object with metadata? It would be good to know what Accept headers result in different behavior when you include them

Reply:
Agreed here!

Reply:
Added:

If no Accept header is passed in the request then the default is to return application/json with metadata. The output field is used to determine whether a GeoJSON or CSV response is returned in the results field of the response.